### ISSUE TITLE
Logo and Ko-fi images missing alt text

### ISSUE THREAD

**Issue by corrinachow** [fariapp/fari#120]
## 🚩 Describe the bug

Icon and Ko-fi images missing alt text

## 🔢 Steps to reproduce

1. Inspect elements for Fari and Ko-fi logos
2. See that alt-text is missing

## 🟢 Expected behavior

Images should have alt text to describe their function

## 🔴 Actual behavior

Images don't have alt text

## 💅 Screenshots

Fari:
![image](https://user-images.githubusercontent.com/35117708/93732466-759fcc00-fb9f-11ea-9560-541a0c4fa28c.png)

Ko-fi:
![image](https://user-images.githubusercontent.com/35117708/93732455-67ea4680-fb9f-11ea-90ba-81c2dca8be73.png)

## 🖥 Desktop or Smartphone
Yes.

- Browser: [e.g. chrome, safari]

## 📄 Additional context

https://webaim.org/techniques/alttext/#basics

**Comment by RPDeshaies** [comment]
I don't control the Kofi image but I can update Fari's icon alt text

**Comment by RPDeshaies** [comment]
will be fixed in #113
